app/src/main/java/com/garethevans/church/opensongtablet/StageMode.java

```diff
@@ -1825,26 +1890,32 @@ private void setupPageButtons() {
             scrollUpButton = findViewById(R.id.scrollUpButton);
             setBackButton = findViewById(R.id.setBackButton);
             setForwardButton = findViewById(R.id.setForwardButton);
-            setUpPageButtonsColors();
+            // IV - minor name change setUp -> setup
+            setupPageButtonsColors();
             setupQuickLaunchButtons();
         });
 
-
-
-
-
         // Decide if we are grouping / tidying page buttons
         groupPageButtons();
 
         // Set the sizes and the alphas
         pageButtonAlpha("");
 
+        // IV - Animations for grouped buttons below removed as they fire in the wrong place
+        // IV - that is on next return to group, perhaps a long time after use
+        // IV - the collapse of the group will 'animate' the click
         // Set the listeners
         setButton.setOnClickListener(view -> {
             CustomAnimations.animateFAB(setButton,StageMode.this);
             FullscreenActivity.whattodo = "editset";
             openFragment();
         });
+        // IV - New Long click to set item variation
+        setButton.setOnLongClickListener(view -> {
+            FullscreenActivity.whattodo = "setitemvariation";
+            openFragment();
+            return true;
```

Comment:
Like this option.  However, this also fires the page gesture (long pressing on a song on the page adds it to the set).
This brings up the add variation, but also adds new song to set.  Because the set popup shows first, the new song added doesn't show.

Reply:
Actually ignore that!  This happens with all the page buttons - it is possible to long click them and the page together with a fat finger!  Might have to increase the padding around the buttons to capture button presses on their own.

Reply:
Strange, this will happen for any long press button action?  I added an ignore section around button use to try to avoid main screen mis-fires!

I think main screen long press gesture is a big problem - it fires when not wanted a lot.  Fingers are not precise!  It may interfere with dragging in autoscroll etc.  My vote is to remove it (I tend to turn it off when using live) as it fires a little too often when using buttons etc.  Particularly as we like button long press actions!

Reply:
I personally like the long press gesture.  However, might be sensible to set the default behaviour for new users to off?

Reply:
It need to not mis-fire.  I have had it fire during testing of dragging autoscroll.  The gesture could be turned off in autoscroll as dragging can lead to accidental long press.  Must though only fire if well away from buttons or top bar icon / features.   Not sure how you get there :-)  Perhaps restrict it to the area below top bar, below sticky x cancel, above bottom buttons and to right of other buttons. Fun.

Reply:
Changed the default behaviour to off in my branch.  Will push changes once pull request completed